packages/manager/src/features/CloudPulse/Alerts/AlertsResources/ShowTextWithExtraInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Chip, Tooltip, Typography } from '@linode/ui';
+import React from 'react';
+
+import NullComponent from 'src/components/NullComponent';
+
+export interface TextWithInfoProp {
+  /**
+   * The list of texts that needs to be displayed with chip and tooltip setup
+   */
+  values?: string[];
+}
+
+export const ShowTextWithExtraInfo = ({ values }: TextWithInfoProp) => {
```

Comment:
```suggestion
export const TextWithExtraInfo = ({ values }: TextWithInfoProp) => {
```

much better name for a component